wsproxy.ml

```diff
@@ -91,6 +91,7 @@ let handler sock msg =
       Lwt_unix.close sock
 
 let _ =
+  Random.self_init ();
```

Comment:
I think this slipped when partially reverting https://github.com/xapi-project/wsproxy/commit/deb02fdbfb1bf4a01075dd514da67dea920f35b7 however I am merging this given that has already been tested and will eventually remove this line in the other PR if it turns out that it is really not needed